samples/unit-tests/series/stacking/demo.js

```diff
@@ -699,4 +699,46 @@ QUnit.test('Date objects as X values, column', function (assert) {
             'Series 3 - Point should start from value=1 (#4024)'
         );
     });
+
+    QUnit.test(
```

Comment:
No chance to reuse the existing test?

Reply:
The config to recreate the issue is quite specific and I was not able to reuse any of the existing tests.